model/riscv_sys_regs.sail

```diff
@@ -902,6 +916,9 @@ function clause write_CSR(0x7a0, value) = { tselect = value; tselect }
 
 /*
  * Entropy Source - Platform access to random bits.
+ * WARNING: This function currently lacks a proper side-effect annotation.
+ *          If you are using theorem prover tool flows, you
+ *          may need to modify or stub out this function for now.
```

Comment:
Was this intended to be added here? I vaguely recall removing it so maybe it is a rebase mistake?